app/frontend/src/views/form/Manage.vue

```diff
@@ -1,141 +1,232 @@
 <template>
   <div>
-    <v-breadcrumbs :items="breadcrumbs" />
```

Comment:
Should we be getting rid of all breadcrumbs?

Reply:
Yeah there's a ticket to remove them all for initial release. SHOWCASE-1639

Reply:
should've known.  cool.